addons/website_slides/static/src/js/public/components/slide_quiz_finish_dialog/slide_xp_progress_bar.xml

```diff
@@ -16,6 +11,11 @@
                t-out="state.rankLowerBound"/>
         <small t-if="state.rankUpperBound" class="float-end fw-bold"
                t-att-class="{'d-none': state.hideRankBounds}" t-out="state.rankUpperBound"/>
+        <div class="tooltip fade show bs-tooltip-top position-relative" t-att-class="{'level-up': props.levelUp}" role="tooltip"
+             t-attf-style="left: #{state.rankProgressPercentage}%; top: -12px; max-width: 3rem">
```

Comment:
Hello @pyduf :slightly_smiling_face: 
Maybe `top: -14px` would be better here to match the 17.2 display where the arrow was slightly above the bar not directly on the bar?

![image](https://github.com/odoo/odoo/assets/107679666/d1ad5677-3217-4da5-84e5-e962ca50b578)
![image](https://github.com/odoo/odoo/assets/107679666/f27cf999-4ca2-44c8-b0ef-ec20088e1f88)

Also I think this PR should be retargeted to 17.3 :slightly_smiling_face: 
Cheers!

Reply:
@pyduf Hello! :slightly_smiling_face: 
If you rename the branch I think it will automatically create a new PR and a new runbot as well :slightly_frowning_face: 
It's fine like this :slightly_smiling_face:  this way we can keep comments history :slightly_smiling_face: 
Thanks for updating!